unified-runtime/source/adapters/cuda/device.cpp

```diff
@@ -505,7 +505,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
     // OpenCL's "local memory" maps most closely to CUDA's "shared memory".
     // CUDA has its own definition of "local memory", which maps to OpenCL's
     // "private memory".
-    if (hDevice->maxLocalMemSizeChosen()) {
+    if (hDevice->getMaxChosenLocalMem()) {
```

Comment:
I think writing `hDevice->getMaxChosenLocalMem() > 0` instead states the intent a bit better when looking at the line of code in isolation because the current check doesn't explicitly mean `> 0` if one didn't check that `getMaxChosenLocalMem()` returns an `unsigned` integer. I am okay either way and there may be an argument for the brevity here, so just a personal suggestion.

Reply:
Ah I see the idea.
I also missed the fact that 0 is actually valid here.

Reply:
I spoke too fast, actually `0` is invalid it's just the error message in the test that wasn't matching anymore.